fix_hashes.py

```diff
@@ -0,0 +1,192 @@
+# This is just a fix script
+# Do not merge with this file on PR
```

Comment:
Is this right? 🤔

Reply:
yes this shouldn't be merged, we can take it off or use it to complete fixes in this PR then delete it

Reply:
because some of the solution as you see in fix_script output, need faster machines to compile a hash their solution

Reply:
Well doesn't it fulfilled its purpose when used once on the entire repo? Why should it be on this commit as it shouldn't be merged? I'm missing something 😦

Reply:
I understand what are you saying @A-Hilaly. I think we don't need merge this file on the end too. We need only merge this PR when we fix all the hashs, because this a breaking change. So we need to fix all the hashs, so then delete this util script. We don't will need that again after this. And if will, will be on the git history.

Thanks for your job until here @A-Hilaly, awesome.